res/css/views/rooms/_MessageComposer.scss

```diff
@@ -37,6 +37,8 @@ limitations under the License.
     }
 
     .mx_MessageComposer_row {
+        /* This forces the composer to wrap its content, even if it has no word boundaries. Fixes #22507 */
```

Comment:
We don't tend to put issue fix numbers (and even then we normally use full links) in comments unless a lot of additional context is needed, for content wrapping in the composer I don't think much context is needed.